public/static/partials/github_commit_panel.html

```diff
@@ -5,16 +5,16 @@
       <strong ng-bind-html="commit.message | escapeHtml | jiraLinkify: jiraHost | ansi"></strong>
     </div>
     <div class="commit-message-author">
-      <span class="semi-muted">
+      <span ng-if="commit.author" class="semi-muted">
         [[commit.author]]
         ([[commit.author_email]])
       </span>
       <span class="muted">
-        authored at [[(commit.ingest_time || commit.create_time) | convertDateToUserTimezone:timezone:"MMM D, YYYY h:mm:ss a"]]
+        [[ commit.author ? 'authored' : 'Created' ]] at [[(commit.ingest_time || commit.create_time) | convertDateToUserTimezone:timezone:"MMM D, YYYY h:mm:ss a"]]
```

Comment:
nit: consistent capitalization 'authored', so 'created'

Reply:
Created is the start of the sentence here, whereas a name starts the sentence in the other scenario